examples/Pipe-Flow/input.i3d

```diff
@@ -0,0 +1,141 @@
+
+! Mesh
+nx=8                  ! X-direction nodes
+ny=160                ! Y-direction nodes
+nz=160                ! Z-direction nodes
+istret = 0            ! y mesh refinement (0:no, 1:center, 2:both sides, 3:bottom)
+beta = 0.0            ! Refinement parameter (beta)
+
+! Domain
+xlx = 1.25            ! Lx (Size of the box in x-direction)
+yly = 1.6             ! Ly (Size of the box in y-direction)
+zlz = 1.6             ! Lz (Size of the box in z-direction)
+
+! Flow parameters
+re  = 5300.           ! nu=1/re (Kinematic Viscosity)
+
+! Time stepping
+dt = 0.0004           ! Time step
+ifirst = 1            ! First iteration
+ilast = 100           ! Last iteration
+
+! Enable modelling tools
+ilesmod=0             ! if 0 then DNS
+numscalar = 1         ! How many scalars? (Set to zero to disable scalars)
+iibm=2                ! Flag for immersed boundary method
+
+! Boundary and initial conditions
+iin = 1               ! Inflow conditions (1: classic, 2: turbinit, 4: SEM)
+u1  = 1.              ! u1 (max velocity) (for inflow condition)
+u2  = 1.              ! u2 (min velocity) (for inflow condition)
+init_noise = 1.0      ! Turbulence intensity (1=100%) !! Initial condition
+inflow_noise = 0.0    ! Turbulence intensity (1=100%) !! Inflow condition
+
+nclx1 = 0
+nclxn = 0
+ncly1 = 0
+nclyn = 0
+nclz1 = 0
+nclzn = 0
+
+/End
+
+!====================
+&NumOptions
+!====================
+
+! Spatial derivatives
+ifirstder = 4           ! (1->2nd central, 2->4th central, 3->4th compact, 4-> 6th compact)
+isecondder = 4          ! (1->2nd central, 2->4th central, 3->4th compact, 4-> 6th compact, 5->hyperviscous 6th)
+ipinter = 2             ! interpolation scheme (1: classic, 2: optimized, 3: optimized agressive)
+
+! Time scheme
+itimescheme = 3         ! Time integration scheme (1->Euler,2->AB2, 3->AB3, 4->AB4,5->RK3,6->RK4, 7-->CN2+AB3)
+
+! Dissipation control
+nu0nu = 3.0             ! Ratio between hyperviscosity/viscosity at nu
+cnu = 0.44              ! Ratio between hypervisvosity at k_m=2/3pi and k_c= pi
+
+/End
+
+!=================
+&InOutParam
+!=================
+
+! Basic I/O
+irestart = 0          ! Read initial flow field ?
+icheckpoint = 100     ! Frequency for writing backup file
+ioutput = 25          ! Frequency for visualization
+ilist = 25            ! Frequency for writing to screen        
+nvisu = 1             ! Size for visualisation collection
+
+/End
+
+!=================
+&Statistics
+!=================
+
+wrotation = 0.12      ! rotation speed to trigger turbulence
+spinup_time = 5000    ! number of time steps with a rotation to trigger turbulence
+nstat = 1             ! Size arrays for statistic collection
+initstat = 50         ! Time steps after which statistics are collected
+
+/End
+
+!########################
+! OPTIONAL PARAMETERS
+!#######################
+
+!================
+&ibmstuff
+!================
+
+ra=0.5        ! when simulating a cylinder or a pipe, radius
```

Comment:
rai

Reply:
Definitely, the input file (laminar simulation has been updated) in 350f8b4513cdf6400cde9db3c99a8f2ea8ac184b.